### ISSUE TITLE
[Suggest] optimise render algo for custom big models

### ISSUE THREAD

**Issue by CrazyWords1** [sp614x/optifine#5609]
Hi, i'm love to create huge 3d models, with 1k+ elements model in BlockBench.
I ran into a problem, in big models, fps reduced by 90%, but in other 3d engines, these models are the simplest and have almost no effect on fps, i don't understand anything about render algo and other, but i'm know, optifine developers is very smart.

Please think of something)
![изображение](https://user-images.githubusercontent.com/32343185/116785353-983d8000-aab2-11eb-901c-2661ee88b994.png)

**Comment by CrazyWords1** [comment]
I want to do beautiful things, but I meet with such problems, and sorry for my English

**Comment by KaiAF** [comment]
I think this is normal? I am not sure if this is actually an issue with optifine itself

**Comment by CrazyWords1** [comment]
Yes, of course, but optifine make our lifes easier, right? Gives us new opportunities, Maybe the ability to use large models will be one of them?)

**Comment by Jiingy** [comment]
Closing this issue as it is a very big ask, with little to no details on how specifically to execute the optimizations desired. If you are able to come up with specific optimizations that can realistically be made, we would love to hear it!
